source/syntax_tables.patch

```diff
@@ -137,8 +141,6 @@ index 0000000..85bca89
 +  const OperandParameters& operands = inst.operands;
 +  for (int i = 0; i < operands.getNum() ; i++) {
 +    switch (operands.getClass(i)) {
-+      case spv::OperandOptionalId:
-+      case spv::OperandOptionalImage:
```

Comment:
Yes, they were removed from the doc.h in the SPIR-V spec generator.
So they didn't appear in the grammar and would be dead enums if not removed here.